src/Models/ApplicantsReportsModel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Portal\Models;
+
+use PDO;
+use Portal\Interfaces\ApplicantModelInterface;
+
+class ApplicantsReportsModel
+{
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function extractDataForApplicantReports(int $reportType, int $startDate, int $endDate): array
+    {
+        switch ($reportType) {
+            case 1:
+                $report = 'gender';
+                $column = '`gender`.`gender`';
+                $table = "gender";
```

Comment:
can we use '`gender`' for code consistency, same for the other $table variables below. Single quotes should work fine I think

Reply:
All double quotes have been replaced with single quotes!